### PR TITLE
Add View/Color menu to switch colors for greyscale images

### PR DESCRIPTION
**Pull request by rbnvrw**
Add the option to select a color from the menu for greyscale images.

### CODE REVIEW

File: pimsviewer/utils.py [hunk 1]
```diff
@@ -56,6 +56,13 @@ def to_rgb_uint8(image, autoscale=True):
     else:
         raise ValueError("No display possible for frames of shape {0}".format(shape))
 
+    if grayscale:
```
Comment (caspervdw) [inline pimsviewer/utils.py]: You can make use of `pims.to_rgb(image, [force_color])`, possibly at line 30?

File: pimsviewer/viewer.py [hunk 2]
```diff
@@ -817,3 +853,8 @@ def export_video(self, filename=None, rate=None, **kwargs):
         export(pipeline(to_rgb_uint8)(self.reader), filename,
```
Comment (caspervdw) [inline pimsviewer/viewer.py]: Please add the autoscale and force_color kwargs here too

File: pimsviewer/viewer.py [hunk 3]
```diff
@@ -729,17 +747,35 @@ def to_clipboard(self):
...
+    def update_color_mode(self, color=None):
+        """Updates the color mode (full color/greyscale) of the current image.
+        """
+        self.force_color = self._available_colors[color]
+        self.update_view()
 
-    # def to_frame(self):
-    #     return Frame(rgb_view(self.to_pixmap().toImage()),
-    #                  frame_no=self.index['t'])
+    def _generate_color_menu(self):
```
Comment (caspervdw) [inline pimsviewer/viewer.py]: Is there a reason to put this in a separate function? Do we want to regenerate the color menu at some point?
Else I'd prefer putting this in the `__init__`.

Comment (rbnvrw) [inline pimsviewer/viewer.py]: Just for code clarity and to prevent the `__init__` from becoming very long.
But I moved it to the init function.

**Comment by rbnvrw** [comment]
I updated the code with your suggestions.

**Pull request merged**